scratch/camera_feed/display_camera_feed.cpp

```diff
@@ -0,0 +1,241 @@
+#include "../../src/network/network.hpp"
+#include <stdlib.h>
+#include <string.h>
+#include <iostream>
+#include <turbojpeg.h>
+#include <SDL2/SDL.h>
+//#include <inttypes.h> to print uints
+// If you are trying to print a uintxx_t there are 2 methods I have found.
+/*
+    std::cout << unsigned(uint you are trying to print) << std::endl;
+    printf("%" PRIu16 "\n", uint you are trying to print);
+ */
+typedef struct BufferItem_s{
+    uint16_t frame_index;
+    uint16_t data_size;
+    uint8_t sections_remaining;
+    unsigned char* data_sections;
+}BufferItem;
+
+const int CAMERA_MESSAGE_FRAME_DATA_MAX_SIZE  = 65000;
+const int CAMERA_FRAME_BUFFER_SIZE = 6220800;
+const int CAMERA_FRAME_BUFFER_COUNT = 5;
+/*
+ This will change the size of the window rendered.
+ Standard Resolution is 1920 by 1080
+ */
+const int WINDOW_X_LENGTH = 1920;
+const int WINDOW_Y_LENGTH = 1080;
+const long unsigned int JPEG_SIZE = WINDOW_X_LENGTH * WINDOW_Y_LENGTH * 3;
+const char* DESTINATION_ADDRESS = "127.0.0.1";
+const tjhandle jpeg_decompressor = tjInitDecompress();
```

Comment:
make this non-const, and call tjInitDecompress at runtime in the main method. That way, you can do error checking in the main method, once. Then you can remove the if block at line 34.